src/views/accounts/AccountChangeSecret/Overview/CardSummary.vue

```diff
@@ -47,7 +47,7 @@ export default {
     LogItems() {
       return [
         {
-          title: this.$t('Tasks'),
+          title: this.$t('Total'),
           body: {
             count: this.data.total_count_change_secrets
           }
```

Comment:
In this code snippet, there is no difference in syntax or structure. It appears it might be a reference to different strings within the application's localizable text format.

Therefore, based on these findings, there are no differences found regarding code formatting here between current day (2025) and past days. If you need help with something else, feel free to ask!